ext/native/ext/cityhash/city.cpp

```diff
@@ -87,18 +88,19 @@ static uint32 UNALIGNED_LOAD32(const char *p) {
 #endif
 
 #else
+// TODO: Why is this path being built on Android? Slow.
```

Comment:
Should we remove this comment now (I think the `|| defined(ANDROID)` is newer?)  By the by, was it using the wrong path before too?  I guess yes?

-[Unknown]